src/SmartComponents/ErrorAlert/ErrorAlert.js

```diff
@@ -1,90 +1,54 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
 import { connect } from 'react-redux';
-import { withRouter } from 'react-router-dom';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
 
-import { compareActions } from '../modules';
-import { errorAlertActions } from './redux';
-import { setHistory } from '../../Utilities/SetHistory';
-import { baselinesTableActions } from '../BaselinesTable/redux';
-
-class ErrorAlert extends Component {
+export class ErrorAlert extends Component {
     constructor(props) {
         super(props);
     }
 
-    confirmDriftModal = () => {
-        const { toggleModal, revertCompareData, history, previousStateSystems } = this.props;
-
-        toggleModal();
-        revertCompareData();
-        setHistory(history, previousStateSystems.map(system => system.id));
-    }
-
-    confirmBaselineModal = () => {
-        const { toggleModal, revertBaselineFetch, tableId } = this.props;
+    closeToast = (tableId) => {
+        const { onClose } = this.props;
 
-        toggleModal();
-        revertBaselineFetch(tableId);
+        if (tableId) {
+            onClose(tableId);
+        } else {
+            onClose();
+        }
     }
 
     render() {
-        const { errorAlertOpened, error, baselineError } = this.props;
+        const { addNotification, error, tableId } = this.props;
 
         return (
             <React.Fragment>
-                { errorAlertOpened && (
-                    <Alert
-                        variant="danger"
-                        title="Error"
-                        action={
-                            <AlertActionCloseButton
-                                onClose={ () =>
-                                    error.status ? this.confirmDriftModal() : this.confirmBaselineModal()
-                                }
-                            />
-                        }
-                    >
-                        Status Code: { error.status ? error.status : baselineError.status }<br></br>
-                        { error.detail ? error.detail : baselineError.detail }
-                    </Alert>
-                ) }
+                { error.status
+                    ? addNotification({
+                        variant: 'danger',
+                        title: 'Error',
+                        description: error.detail,
+                        dismissable: true
+                    })
+                    : null
+                }
+                { error.status ? this.closeToast(tableId) : null }
```

Comment:
I'm a bit confused. :) Does this component render something or is it only to call functions when `error.status` is set?

Reply:
This component calls the `addNotification` in frontend-components/notifications whenever the error object that is passed in has a status code. I don't pass an error with a status of 200, so it would only ever get an error status.